packages/davinci-client/src/lib/client.store.ts

```diff
@@ -317,9 +317,13 @@ export async function davinci<ActionType extends ActionTypes = ActionTypes>({
         return handleUpdateValidateError('Collector not found', 'state_error', log.error);
       }
 
-      if (collectorToUpdate.category !== 'ValidatedSingleValueCollector') {
+      if (
+        collectorToUpdate.category !== 'ValidatedSingleValueCollector' &&
+        collectorToUpdate.category !== 'ObjectValueCollector' &&
+        collectorToUpdate.category !== 'MultiValueCollector'
+      ) {
         return handleUpdateValidateError(
-          'Collector is not a SingleValueCollector and cannot be validated',
+          'Collector is not a SingleValueCollector, ObjectValueCollector, or MultiValueCollector and cannot be validated',
```

Comment:
I'm wondering if we're reaching a tipping point with these error messages. We may want to consider rewording these as its possible we have to add more collectors that will fit this error.

Reply:
I was thinking this too. Maybe something like `Collector does not fall into a category that can be validated`

Reply:
Let's hope we don't have to invent new collector categories :D